tests/test_loraplus_helper.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
```

Comment:
Oh, just noticed the file name, let's remove the `_helper` part, as it's not necessary.